spec/problem_one_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../problem_one'
+
+describe ProblemOne do
+  it 'computes for a set of even numbers' do
+    even_set = [2, 8, 12, 10, 128, 64, 32]
```

Comment:
Could you use `let` in your tests?

Recommended:
https://www.bignerdranch.com/blog/rspec-tip-let-and-nested-describecontext-blocks/
https://relishapp.com/rspec/rspec-core/v/3-8/docs/subject/explicit-subject

Reply:
I'm having a tough time applying `let`, `subject` and the three-phase pattern after placing the functions outside classes. Do they all have to be used at the same time?

Three-phase pattern: https://www.justinweiss.com/articles/writing-better-tests-with-the-three-phase-pattern/